src/main/java/seedu/address/storage/JsonAdaptedPatient.java

```diff
@@ -27,6 +27,7 @@ class JsonAdaptedPatient {
     private final String icNumber;
     private final String birthday;
     private final String address;
+    private final String department;
```

Comment:
Would it better to follow the naming convention we have, ie `assignedDepartment`